ruby/README.md

```diff
@@ -827,6 +824,179 @@ class TestClass
 end
 ```
 
+### SOLID Object Oriented Design
```

Comment:
🔧⛏️ This references [SOLID](https://en.wikipedia.org/wiki/SOLID), but the only topic actually discussed is `Single Responsibility Principle`, so it's not clear if the `OLID` topics are supposed to be adhered to. I'd recommend either changing this heading, adding a description that outlines expectations for `SOLID` more clearly, or adding sections and examples for the `OLID` topics.

Reply:
There's a section just below this header that informs the reader to check out the wiki page (which is all we used to have) -- if you'd like, I could improve the heading on line 830 to instead say something like: "Single Responsibility Principle in More Detail"?